packages/harvester-cloud-provider/package.yaml

```diff
@@ -0,0 +1,3 @@
+url: https://github.com/harvester/charts/releases/download/harvester-cloud-provider-0.1.3/harvester-cloud-provider-0.1.3.tgz
+packageVersion: 00
+releaseCandidateVersion: 00
```

Comment:
dev-v2.6 does not utilize `packageVersion` or `releaseCandidateVersion`

Can you switch this to `version: 100.0.0`?